public/app/percona/backup/components/BackupInventory/RestoreBackupModal/RestoreBackupModal.tsx

```diff
@@ -63,6 +166,27 @@ export const RestoreBackupModal: FC<RestoreBackupModalProps> = ({
                   disabled={values.vendor !== DATABASE_LABELS[Databases.mysql]}
                 />
                 <TextInputField disabled name="vendor" label={Messages.vendor} />
+                {backup!.mode === BackupMode.PITR && (
```

Comment:
I would use `backup?` instead, as `!` will force this to pass even if it's undefined or null

Reply:
Fixed. Probably miss click.